buildroot/package/rpi-firmware/config.txt

```diff
@@ -5,3 +5,4 @@ fixup_file=fixup_rc.dat
 [pi4]
 start_file=recover4.elf
 fixup_file=fixup4rc.dat
+max_framebuffers=2
```

Comment:
I googled and found https://www.raspberrypi.org/forums/viewtopic.php?t=245789 which says that `max_framebuffers` has to match the number of physical displays. So does this 'break' things when only a single display is attached? :shrug:

Reply:
That answer of "matching" was not confirmed by JamesH. 
AIUI, it is the **maximum** number of framebuffers that could be allocated, so it can be greater than or equal to the number of displays, although setting it greater might be  a waste of resources.
Certainly if it is less than the number of displays, you won't get the framebuffer.
You might be able to use HDMI1 with only 1 framebuffer if you set framebuffer_priority, but I've not tried it, but that would also mean the user needs to set it in config.txt, rather than NOOBS dynamically using which ever is connected. So you need to set max_framebuffers=2 to access HDMI1 with this technique.

Reply:
> it can be greater than or equal to the number of displays, although setting it greater might be a waste of resources.

Ah cool. Obviously "wasting resources" could be an issue on Raspberry Pi OS, but isn't an issue for a limited environment like NOOBS. Sounds like you've already tested it thoroughly :+1: 
I hope you didn't mind me asking!

Reply:
I always welcome your pedantic mindset! 😄
It's already been released in PINN and seems to work quite well. I think it had the most downloads immediately after a release, so users must like the feature.
But all the comments here have enabled me to improve things further (especially the builds) and I've learnt some more stuff along the way. So thanks to all.

Reply:
FYI: https://github.com/raspberrypi/documentation/commit/4f691ff4cc96eb938631caf5924005eb8817a401

Reply:
Like I said.

Reply:
u guys should read the datasheets first, I know they are primitive. 

But you could ask for help. I feel James got the production ones 😏

PS - James, transfer it to a copy with the notes. Don't tell them about the piece of scrap. They cannot Google it.